components/HomePage/InfoSection.tsx

```diff
@@ -1,21 +1,24 @@
 import styled from 'styled-components'
 
-//todo: add dynamic content based on database class descriptions
-export default function InfoSection({ id, imgStart }) {
+interface InfoSectionProps {
+  id?: string
```

Comment:
made it optional because im using the same interface for the stylecomponent on line 50, which doesnt need the id. I couldnt get Omit to work. Should I separate?

Reply:
you answered this in a lower comment. ignore me.

Reply:
actually i dont even need id as im not using the scrolling anymore :D